latest_cov_build.py

```diff
@@ -14,16 +15,29 @@
 
 name = 'latest.artifacts.public.build.target.tar.bz2'
 
-# Retrieving artifact
-urlretrieve('https://index.taskcluster.net/v1/task/gecko.v2.' +
-            'mozilla-central.latest.firefox.linux64-ccov-opt/' +
-            'artifacts/public/build/target.tar.bz2', name)
+index = taskcluster.Index()
+queue = taskcluster.Queue()
+
+taskId = index.findTask('gecko.v2.mozilla-central.' +
+                        'latest.firefox.linux64-ccov-opt')['taskId']
+
+# Retrieving latest artifact
+url = queue.buildUrl('getLatestArtifact', taskId, 'public/build/target.tar.bz2')
```

Comment:
To avoid rewriting the code multiple times you could:
```Python
for name in ['target.tar.bz2', 'target.code-coverage-gcno.zip', 'chrome-map.json']:
    url = queue.buildUrl('getLatestArtifact', taskId, 'public/build/{}'.format(name))
    urlretrieve(url, name)

 with tarfile.open('target.tar.bz2', 'r:bz2') as tar:
     tar.extractall()
 os.remove('target.tar.bz2')
```

Reply:
Done. :)

Reply:
Oops, noticed an unnecessary line with name declaration. Will change it.